statsdbinterface/database/extmodels.py

```diff
@@ -2,6 +2,7 @@
 from werkzeug.exceptions import NotFound
 from .core import db
 from .models import Game, GamePlayer, GameServer, GameWeapon
+from .models import Mutator as GMut, Mode as GMode
```

Comment:
Class names should be unique across an application, maybe we should rename them?
Urm right, `s/lines/import statements/`, you should use the same statement, like e.g.

```
from ... import a, b, c, \
    d, e, f
```